Tests/Unit/Collector/FormatterTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Http\HttplugBundle\Tests\Unit\Collector;
+
+use Http\Client\Exception\HttpException;
+use Http\Client\Exception\TransferException;
+use Http\HttplugBundle\Collector\Formatter as CollectorFormatter;
+use Http\Message\Formatter;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+class FormatterTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Formatter
+     */
+    private $formatter;
+
+    /**
+     * @var CollectorFormatter
+     */
+    private $subject;
+
+    public function setUp()
+    {
+        $this->formatter = $this->createMock(Formatter::class);
```

Comment:
the phpunit version that supports this is unfortunately php 5.6 and above only. this bundle still supports 5.5 so you need getMock - and if formatter has no interface and constructor arguments, the full getMockBuilder->disableOriginalConstructor->getMock() stuff.